dashboard/app/helpers/district_dropdown_constants.rb

```diff
@@ -0,0 +1,8 @@
+module DistrictDropdownConstants
```

Comment:
You can accomplish that in a number of ways:

1. As I suggested above, add validations to a common module that is included in both enrollment and user models. You'll still need to store the fields in both models, but the validations will be centralized.
2. Split out user_district (or whatever we want to call it) into its own model (and db table), which each user and enrollment will have a foreign key reference to. All the validations will be in the model.
3. Copy the validations between user and enrollment. This might make the most sense if there are significant differences between them, but otherwise it violates DRY.